pkg/client/didexchange/client.go

```diff
@@ -76,7 +76,7 @@ func New(ctx provider) (*Client, error) {
 		return nil, errors.New("cast service to DIDExchange Service failed")
 	}
 
-	connectionStore, err := connectionstore.NewConnectionRecorder(ctx)
+	connectionStore, err := connection.NewConnectionRecorder(ctx)
```

Comment:
This probably could have been connection.NewRecorder

Reply:
done